package.json

```diff
@@ -1,39 +1,25 @@
 {
   "name": "ventuz-director",
-  "version": "1.0.2",
-  "api_version": "1.0.0",
-  "keywords": [
-    "Media Server",
-    "Template Graphics"
-  ],
-  "manufacturer": "Ventuz Technology",
-  "product": "Director",
-  "shortname": "Director",
-  "description": "Companion Module to control Ventuz Director (Enterprise edition version 6.11+)",
+  "version": "1.1.0",
```

Comment:
This module will not be backwards compatible with Companion v2.x, which is considered a breaking change and therefore you should increase the major version.